settings.gradle.kts

```diff
@@ -1,5 +1,6 @@
 plugins {
     id("com.gradle.enterprise") version "3.13.3"
+    id("org.gradle.toolchains.foojay-resolver-convention") version "0.5.0"
```

Comment:
Is it required for CI, or you have just used it for local testing?

Reply:
See https://github.com/gradle-nexus/publish-plugin/pull/225/commits/26441f492ff586b1d79cba044e61b15e34dae0f1